src/__test__/simulator/sdk.ts

```diff
@@ -128,13 +125,7 @@ class sdk2 {
 					}
 					if (conversation) {
 						const message = conversation.message;
-						if (!isApproved && this.uid) {
```

Comment:
Why we remove this, we can create corner test cases for isApproved case when needed.

Reply:
Because it is the parameter input that we pass in. It can't guarantee anything or cover a case in real scenario